files/default/csr.py

```diff
@@ -328,4 +416,18 @@ def setup_logging(log_file, max_log_size, logLevel):
             except Exception, e:
                 LOG.error("Error while rotating key: {0}".format(e))
                 raise e
+    elif args.operation == "elkadmin":
+        LOG.debug("ELK Admin certificate")
+        cert = ELKAdminCertificate(config)
+        
+        cert.create_csr()
+        
+        h = Host(config, cert, state_store)
+        with requests.Session() as session:
+            try:
+                h.generate_elkadmin_cert(session)
+                subprocess.call(config.keystore_script)
```

Comment:
Why do you need to call the bash script here?

Reply:
Fixed